packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -52,6 +64,17 @@ Therefore, when dealing with Custom Roles, these three entities always exist:
 * A principal: Either a Group or a User - known as a Member of the Binding
 * A Resource Set: Identified by its `id`
 
+### Custom Role Assignment for IAM-based Standard Roles
+<ApiLifecycle access="ea" />
+When using IAM-based Standard Roles the same concepts as Custom Roles apply with the following distinctions:
+1. IAM-based standard roles can only be used with pre-defined Resource Sets.
+2. For both IAM-based Standard Roles and Resource Sets, there is a predefined constant `id`. This `id` will always be the `type` of the Role or Resource Set.
+
+| Role id (type)                                          | Applicable Resource Set id (type)    |
+| :------------------------------------------------------ | :----------------------------------- |
+| `IGA_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | `OKTA_IGA_CERTIFICATIONS_IAM_POLICY` |
```

Comment:
@ryanbradley-okta I know you advocated to remove 'OKTA_' from the standard IAM roles. There is a non-zero chance we'll show the resource sets here in the UI in the future, would you want OKTA_ removed from them as well if/when they are displayed to customers?

Reply:
@bhamel-okta Yes, we should remove the 'OKTA_' prefix if this may be seen by customers.